tests/greenpool_test.py

```diff
@@ -7,11 +7,6 @@
 import tests
 
 
-def passthru(a):
```

Comment:
this function is used below, see line 29.

Reply:
There were two `passthru` functions, there's another one defined later on.

Reply:
lol :)

As the other is defined later, I suppose that this is this other one which is loaded on memory at runtime.